hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -341,7 +341,7 @@ public void onEvict(Data key, R record) {
 
     protected void invalidateEntry(Data key, String source) {
         if (isInvalidationEnabled()) {
-            cacheService.sendInvalidationEvent(name, toHeapData(key), source);
+            cacheService.sendInvalidationEvent(getName(), toHeapData(key), source);
```

Comment:
It seems to be a leftover from an approach to introducing provisional cache names that was later discarded. Seems I'm not overriding `getName()` anymore so we can revert this to `protected final String name`.